nano/node/node.cpp

```diff
@@ -702,6 +705,7 @@ void nano::node::keepalive_preconfigured (std::vector<std::string> const & peers
 {
 	for (auto i (peers_a.begin ()), n (peers_a.end ()); i != n; ++i)
 	{
+		// Question: why isn't keepalive being called with `network.port`? looks like it should. Can we make this change?
```

Comment:
I want to push another commit to either fix this or at least remove the comment after we've discussed it.

@clemahieu @dsiganos -- does this look ok to you? Why would the default value be used for these keepalives?

Reply:
The default values is used for the preconfigured keepalives because the preconfigured peers config only takes an address and not address and port pair as it should.

Reply:
Ohh okay so we can't use our port because they use the default one anyways...if they haven't been set up with another explicit one which they mustn't in order for this to work... Ok I see. Well, yeah I agree with you there's some room for more flexibility in there, but it's clear for me now so I'll just take out the question and leave a small comment in there.

Reply:
Alright, updated the comment.